base/reflection.jl

```diff
@@ -1525,7 +1525,7 @@ internals.
 - `world::UInt = Base.get_world_counter()`: optional, controls the world age to use
   when looking up methods, use current world age if not specified.
 - `interp::Core.Compiler.AbstractInterpreter = Core.Compiler.NativeInterpreter(world)`:
-  optional, controls the abstract interpreter to use, use the native interpreter if not specified.
+  optional, controls the abstract interpreter to use, use The abstract interpreter if not specified.
```

Comment:
I don't understand the intention of this change.

Reply:
Thanks, remnants of find and replace.